src/main/java/com/google/sps/servlets/Post.java

```diff
@@ -0,0 +1,160 @@
+/* Post Object:
+Holds the information in the cards
+- Post ID
+- Organization Name
+- The date (month, day, year)
+- The start time (hour, minute)
+- The end time (hour, minute)
+- location (lat, long)
+- Number of people it feeds
+- Type of food
+- Description
+- College Id
+*/
+
+package com.google.sps.servlets;
+import javax.servlet.http.HttpServletRequest;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.PreparedQuery;
+import java.util.ArrayList;
+import java.util.TimeZone;
+import java.util.Calendar;
+
+public class Post {
+    String postId = "";
+    String organizationName = "";
+    int month = 0;
+    int day = 0;
+    int year = 0;
+    int startHour = 0;
+    int startMinute = 0;
+    int endHour = 0;
+    int endMinute = 0;
+    String location = "";
+    Double lat = 0.0;
+    Double lng = 0.0;
+    String numberOfPeopleItFeeds = "";
+    String typeOfFood = "";
+    String description = "";
+    String collegeId = "";
+    int timeSort = 0;
+
+    public Post() {
+        // All variables are already initialized.
+    }
+
+    /* Fill in the important Post details from the POST request. */
+    public void requestToPost(HttpServletRequest request, String collegeId) {
```

Comment:
For methods like this, I would like to see tests eventually.

Reply:
Sounds good!